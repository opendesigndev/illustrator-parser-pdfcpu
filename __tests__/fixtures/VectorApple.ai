version https://git-lfs.github.com/spec/v1
oid sha256:c5fc34346629c3379cee6065c45b7de0b620be822604551d1463d8e691b03d06
size 1062861
