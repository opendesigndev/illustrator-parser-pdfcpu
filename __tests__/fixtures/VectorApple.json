version https://git-lfs.github.com/spec/v1
oid sha256:c3439fe43eec5a69abdd981abbe26e16b8f6269416db4f1af1ff663e46e8c5f5
size 377823
