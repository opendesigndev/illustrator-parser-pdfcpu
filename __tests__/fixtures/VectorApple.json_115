version https://git-lfs.github.com/spec/v1
oid sha256:5fb3eed567466e7dbe8a1d490442b3225c4c1f868fb46b707392c822d01521f5
size 177
