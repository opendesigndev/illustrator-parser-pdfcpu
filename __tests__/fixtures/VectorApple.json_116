version https://git-lfs.github.com/spec/v1
oid sha256:5bb37cb80fe6b028a8b76cda1dadc59d6ceb98c9a1615cac82f6b93f408d2b5c
size 176
