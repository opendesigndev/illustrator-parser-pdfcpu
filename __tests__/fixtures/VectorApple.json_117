version https://git-lfs.github.com/spec/v1
oid sha256:e1fbdfa88d189d7f479928728503441b3894451be8ee032d0231fc3fd30498c1
size 175
