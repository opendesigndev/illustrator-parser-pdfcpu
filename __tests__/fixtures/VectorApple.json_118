version https://git-lfs.github.com/spec/v1
oid sha256:8f3ed523c69a4ccf975e624ed16656fdaa998ad9c267c20bb8a4e90dd398abdd
size 173
