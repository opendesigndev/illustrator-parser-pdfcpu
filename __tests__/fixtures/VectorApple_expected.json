version https://git-lfs.github.com/spec/v1
oid sha256:e9d65b9f31f8a7c2473d32cb600c843c570e872cfc10de71632c960407e7fd03
size 7543004
